### PR TITLE
Refactor README.md

### PR DESCRIPTION
**Pull request by xlk3099**
- Fix spelling typos, e.g: change "simple" to "sample", "infomation" …
- Add descriptions for few APIs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,25 +100,27 @@ The bytes to be hashed (using keccak256) for signing are the concatenation of th
...
+**Sample response**
 
 ```js
 {}
 ```
 
 ## CancelAllOrders
 
-**Request**`POST /v1/cancelallorders`
+Cancel all existing orders using given parameter.
```
Comment (kr1sten0) [inline README.md]: parameter => parameters

File: README.md [hunk 2]
```diff
@@ -274,20 +283,20 @@ http://kovan.dex.top/v1/pairinfo/ETH_ADX
 
 ## GetTradeHistory
 
-Get recent trades by PairId sort by trade time
+Get recent trades by PairId sort by trade time.
```
Comment (kr1sten0) [inline README.md]: PairId => pairId

File: README.md [hunk 3]
```diff
@@ -437,12 +447,12 @@ http://kovan.dex.top/v1/activeorders/0x6a83D834951F29924559B8146D11a70EaB8E328b/
 
 ## GetPastOrders
 
-Get user's filled orders on current wallet address
+Get filled orders on current wallet address.
```
Comment (kr1sten0) [inline README.md]: filled => past

**Comment by kr1sten0** [comment]
Very thx. 👍

File: README.md [hunk 4]
```diff
@@ -1,20 +1,22 @@
 # DEx Open API
 ---
 
-# Account Prepare
+# Account Preparation
 
-1. register a account by email on [https://kovan.dex.top](https://kovan.dex.top)
-2. binding trader address in account page.
+1. register an account by email on [https://kovan.dex.top](https://kovan.dex.top)
+2. bind the trader address on the account page.
```
Comment (libreoscar) [inline README.md]: Thanks for fixing this. Make the same fix in the line blow?

File: README.md [hunk 5]
```diff
@@ -1,20 +1,22 @@
...
 3. deposit eth or tokens in balance page.
 
 # Trade APIs
 
 ## PlaceOrder
 
+Place a new order using given parameters.
```
Comment (libreoscar) [inline README.md]: Just "Place a new order."

File: README.md [hunk 6]
```diff
@@ -75,18 +77,20 @@ The bytes to be hashed (using keccak256) for signing are the concatenation of th
 
 ## CancelOrder
 
-**Request**`POST /v1/cancelorder`
+Cancel an existing order using given parameters.
```
Comment (libreoscar) [inline README.md]: Ditto. Just "Cancel an existing order."

File: README.md [hunk 7]
```diff
@@ -161,18 +169,18 @@ The bytes to be hashed (using keccak256) for signing are the concatenation of th
 
 ## GetMarketInfo
 
-This API for getting important market infomation such as market contract address and token Code for signature
+Get relevant market information such as market contract address and token Code for signature.
```
Comment (libreoscar) [inline README.md]: "token Code" -> "token codes"

File: README.md [hunk 8]
```diff
@@ -212,26 +220,27 @@ http://kovan.dex.top/v1/market
 
 ## GetPairsByCash
 
-This API for getting realtime  trade infomation of  all pair of specific cashtoken such as ETH. This API data can get all tradepair realtime trade infomation.
+Get real-time trade information of all pair of specific cash token such as ETH.
```
Comment (libreoscar) [inline README.md]: Get the real-time trading information of all available trading pairs of the specified cash token (e.g. "ETH").
Remove "This API data can get all trade pair real-time trade information."

File: README.md [hunk 9]
```diff
@@ -245,19 +254,19 @@ http://kovan.dex.top/v1/pairlist/ETH
 
 ## GetPairInfo
 
-Simple TradePair realtime data API by pairId
+Get real-time trade pair data by pairId.
```
Comment (libreoscar) [inline README.md]: Get the real-time information of a trading pair.

File: README.md [hunk 10]
```diff
@@ -320,19 +329,20 @@ http://kovan.dex.top/v1/tradehistory/ETH_ADX/3
 
 ## GetPairDepth
 
+Get pair depth given the trade pair id and the depth size.
```
Comment (libreoscar) [inline README.md]: Get the depth data of a trading pair.

File: README.md [hunk 11]
```diff
@@ -393,26 +403,26 @@ http://kovan.dex.top/v1/depth/ETH_ADX/5
 
 ## GetActiveOrders
 
-Get user's unfilled orders or partical filled orders on current wallet address
+Get unfilled orders or partial filled orders on current wallet address.
```
Comment (libreoscar) [inline README.md]: Get unfilled or partially filled orders that have not been cancelled or expired of a trader.

File: README.md [hunk 12]
```diff
@@ -486,23 +496,23 @@ http://kovan.dex.top/v1/pastorders/0x6a83D834951F29924559B8146D11a70EaB8E328b/ET
 
 ## GetOrderById
 
-Get order of the specified order id
+Get an order details given a specified order id.
```
Comment (libreoscar) [inline README.md]: Get the details of an order by order id.

File: README.md [hunk 13]
```diff
@@ -603,21 +615,23 @@ Reponse:
 
 ## Balance
 
+Get balances of all tokens given the trader address.
```
Comment (libreoscar) [inline README.md]: Get the balances of all tokens of a trader.

**Comment by libreoscar** [comment]
Oh, it's merged already. Never mind.

**Pull request merged**